src/helper.js

```diff
@@ -90,6 +91,10 @@ function getUserLogsPath(name, network) {
 	return path.join(this.HOME, "logs", name, network);
 }
 
+function getStoragePath() {
+	return path.join(this.HOME, "storage");
```

Comment:
Totally agree, but we need to have a global strategy/direction before going heads first. Still beyond the reach of this PR, I think. Until then, I'm good with the current.